### PR TITLE
feature/DED-129 changing screens in monetization and app review

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

For the time being, deploy previews are generated using the Nelify CLI. More info on Slack (#plaform-docs).

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/monetization/app-review-process/index.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ If you want to use the word "LiveChat" in your app name, the only acceptable way
 
 Let's use an example from the <a href="https://developers.livechat.com/console/apps/explore" target="_blank" rel="noopener noreferrer">**App ideas**</a> section in Developer Console.
 
-<img alt="LiveChat App Ideas" src="/images/monetization/app-review-process/livechat-app-ideas.png" width="800px" height="485px"/>
+<img alt="LiveChat App Ideas" src="/images/monetization/app-review-process/App-review-process-app-ideas.png" width="1542px" height="913px"/>
```
Comment (ale-kacperczyk) [inline src/pages/monetization/app-review-process/index.mdx:64]: A few words about image naming:
- names shouldn't start with a capital latter
- it's best for SEO if image names contain the word "livechat"

Comment (kamilatokarska) [reply src/pages/monetization/app-review-process/index.mdx]: done:)

File: src/pages/monetization/how-to-monetize-apps/index.mdx [hunk 2]
```diff
@@ -153,9 +153,16 @@ You can check your current account balance in the **My Revenue** section. You'll
...
-  height="590px"
+  src="/images/monetization/how-to-monetize-apps/livechat-monetization-revenue.png"
+  width="1320px"
+  height="897px"
+/>
+
+<img
+  alt="LiveChat My Revenue Section"
+  src="/images/monetization/how-to-monetize-apps/livechat-monetization-revenue1.png"
+  width="1130px"
```
Comment (ale-kacperczyk) [inline src/pages/monetization/how-to-monetize-apps/index.mdx:164]: Is this the correct size? The image seems a little squeezed https://deploy-preview-1139--livechat-public-docs.netlify.app/monetization/how-to-monetize-apps

Comment (kamilatokarska) [reply src/pages/monetization/how-to-monetize-apps/index.mdx]: You are totally right, I looked too much at all of them and did not notice it. Already changed, should be fine now.

**Review by ale-kacperczyk — APPROVED** [review]
Great job 🎉

**Pull request merged**